tests/ui/struct_flatten.rs

```diff
@@ -9,10 +9,10 @@
 use structopt::StructOpt;
 
 #[derive(StructOpt, Debug)]
-#[structopt(name = "basic", flatten)]
-struct Opt {
-    #[structopt(short)]
-    s: String,
+#[structopt(name = "basic")]
+enum Opt {
+    #[structopt(flatten)]
+    Variant1,
 }
 
 fn main() {
```

Comment:
Simply delete this test along with the corresponding `.stderr`. There's no way to distinguish between a enum variant an struct.

Reply:
This is fine. I replaced it with the one with the error message you provided

Reply:
Hmm, that is not what we testing here. The test is titled as `struct_flatten` so it's supposed to check `flatten ` on top of a struct. If you want to test an enum - rename the file.

Reply:
Good point

Reply:
done

Reply:
The button! It's really painful to see someone types useless messages...